Backpack/src/main/java/net/skyscanner/backpack/dialog/DialogVariables.kt

```diff
@@ -0,0 +1,21 @@
+package net.skyscanner.backpack.dialog
+
+import android.content.Context
+import net.skyscanner.backpack.R
+
+internal class DialogVariables(val context: Context) {
```

Comment:
Can it be DialogProerties instead of variables

Reply:
Can you explain why? In my mind the name variables is more declarative.